qutebrowser/keyinput/modeparsers.py

```diff
@@ -228,19 +228,8 @@ def __init__(self, win_id, parent=None):
         super().__init__(win_id, parent, supports_count=True,
                          supports_chains=True)
         self.read_config('caret')
+        self._supports_count = False
```

Comment:
Actually having support for counts (like `5w`) would be very useful - for most commands you could probably just do something like:

```python
for _i in range (count):
    webview.triggerPageAction(act)
````